app/views/shared/_profile_formfields.html.erb

```diff
@@ -1,16 +1,16 @@
 <div class="row">
   <div class="col-sm-12 col-md-6">
     <%= f.label :picture, t(:picture, scope: 'profiles.form').html_safe %>
-    <% if @profile.picture.present? %>
+    <!-- :image for active storage and :picture for carrierwave -->
+    <% if @profile.image.attached? && @profile.validate(:image_format_size) %>
```

Comment:
doesn't the valitation happend when saved? or is this the same we have at musicwomen?

Reply:
yeah, it's the same like we have at musicwomen